frontend/src/pages/Activities/AddActivity/AddActivity.tsx

```diff
@@ -49,17 +50,26 @@ const AddActivity = ({}: AddActivityProps) => {
     const { data: autocompleteChoices } = useItemTypeAutoCompleteSuggestions(itemTypeSlug)
     const [popoverOpen, setPopoverOpen] = useState(false)
     const [form] = useForm()
+    const {data: item, status: itemStatus} = useItem(token)
+
     if (!itemType) {
         return <Spin />
     }
+    
+    if(token?.startsWith('A')) {
```

Comment:
i know I kind of went back and forth on this a bit.

I think the best version of this is 

`activities/:type` always routes straight to AddActivity - can't be anything else

`activities/:type/:token` routes to idk ActivityWrapper, which is basically just

```
const ActivityWrapper = () => {
    const {token} = useParams()

    return token? (token.startsWith('A')?<ActivityDetail/>:<AddActivity/>):<Spin/>)
}
```

Reply:
will add middleman